webapi/src/main/scala/org/knora/webapi/slice/resources/repo/model/ResourceCreateModels.scala

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright © 2021 - 2024 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.slice.resources.repo.model
+
+import java.time.Instant
+import java.util.UUID
+
+import org.knora.webapi.IRI
+
+final case class ResourceReadyToCreate(
+  resourceIri: IRI,
```

Comment:
*thought*: It would be nice to use `InternalIri` instead of the `String` type alias to `IRI` in these case classes.

At least this way it is a bit more clear that the provided iri should be internal. WDYT?

Reply:
:D Thanks for the effort. If you do not have vi installed maybe you give the online tutorial a chance: https://www.openvim.com/